### PR TITLE
Fixes! Hard coded values on organization list page.

### LINKED ISSUE #237: Hard coded values on organization list page.
**Describe the bug**
There are some hard-coded values while rendering the organization list.

**To Reproduce**

1. Login into the talawa-admin portal as a super admin.
2. You can see the list of organizations (If you created one).  

**Expected behavior**
All the data should be dynamic instead of hard-coded values.

**Actual behavior**
While rendering the organization list, the admins count, members count, created date, and organization location is hard-coded. 

**Screenshots**
image

**Additional details**
To populate created date dynamically, we need to add a new field in the organization schema as a timestamp.
I'm not sure about organization location, because there is no field for this in a schema and it is not required by the user while creating the organization.

### PR DESCRIPTION
**Pull request by asmitsirohi**
**What kind of change does this PR introduce?**

bugfix

**Issue Number:**

Fixes #237 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

*  The admin's count, members count, created date, and organization location is dynamic.
image

* Field for entering location is added.
image

**If relevant, did you update the documentation?**

No

**Summary**

While rendering the organization list, the admins count, members count, created date, and organization location is hard-coded.
But I resolved this problem, now all the data is dynamic instead of hard-coded values.

**Does this PR introduce a breaking change?**

No

**Other information**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
This is a good start. We need to get the code coverage for `OrgList.tsx` to 100%. It is now 46.15%. The 33.33% improvement is good progress. Please see how we can get to 100%

**Comment by palisadoes** [comment]
@yasharth291 @DangaRanga Please review.

File: src/components/SuperDashListCard/SuperDashListCard.test.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ describe('Testing the Super Dash List', () => {
         orgName="Dogs Care"
         orgLocation="India"
         createdDate="04/07/2019"
+        admins="23"
+        members="123"
```
Comment (yasharth291) [inline src/components/SuperDashListCard/SuperDashListCard.test.tsx:16]: is this change really needed if yes why?

**Review by yasharth291 — CHANGES_REQUESTED** [review]

File: src/components/SuperDashListCard/SuperDashListCard.test.tsx (hunk 1 above)
Comment (asmitsirohi) [reply src/components/SuperDashListCard/SuperDashListCard.test.tsx:16]: Yes, these are needed because the motive of this pull request is to remove the hardcoded values on the organization list page, and in hardcoded values, admins count and members are also there.

To resolve this, I need to add admin and member count values dynamically.

Comment (yasharth291) [reply src/components/SuperDashListCard/SuperDashListCard.test.tsx:16]: yeh dynamical nahi h shyd you are testing it for the hardcoded problem i will love to see a way where we can test the dynamical data by dynamical test data

Comment (asmitsirohi) [reply src/components/SuperDashListCard/SuperDashListCard.test.tsx]: @yasharth291 I have made the changes.

**Review by yasharth291 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]
